### PR TITLE
[coverity] Fix 2 coverity errors

### PR DESCRIPTION
**Pull request by Honry**
- CONSTANT_EXPRESSION_RESULT
  - NESTING_INDENT_MISMATCH 

  Impacted tests(approved): new 0, update 4, delete 0
  Unit test platform: Crosswalk for Android 21.51.544.0
  Unit test result summary: pass 4, fail 0, block 0
  https://crosswalk-project.org/jira/browse/CTS-1737

### CODE REVIEW

**Comment by Honry** [comment]
@haoxli, PTAL.

File: webapi/tct-xmlhttprequest-w3c-tests/xmlhttprequest-py/w3c/open-url-multi-window.htm [hunk 1]
```diff
@@ -15,9 +15,10 @@
           var client = new self[0].XMLHttpRequest()
           client.onreadystatechange = function() {
             test.step(function() {
-              if(client.readyState == 4)
+              if(client.readyState == 4) {
                 assert_equals(client.responseText, "bottom\n")
                 test.done()
+              }
             })
           }
```
Comment (haoxli) [inline webapi/tct-xmlhttprequest-w3c-tests/xmlhttprequest-py/w3c/open-url-multi-window.htm]: Please submit the changes to upstream

**Comment by haoxli** [comment]
LGTM

File: webapi/tct-xmlhttprequest-w3c-tests/xmlhttprequest-py/w3c/open-url-multi-window.htm (hunk 1 above)
Comment (Honry) [inline webapi/tct-xmlhttprequest-w3c-tests/xmlhttprequest-py/w3c/open-url-multi-window.htm]: Already done.

**Pull request merged**